src/select/select.js

```diff
@@ -442,7 +442,7 @@ class Select extends React.Component<PropsT, SelectStateT> {
    */
   getValueArray(value: ValueT): Array<OptionT> {
     if (!Array.isArray(value)) {
-      if (value === null || value === undefined) return [];
+      if (value === '' || value === null || value === undefined) return [];
```

Comment:
`value` is typed as an object, not a string. does flow not display an error?

Reply:
Ugh, my bad it’s because of ReduxForm. They type `props.input.value` as `any`, which I was just passing along to Select:
> The value of this form field. It will be a boolean for checkboxes, and a string for all other input types. If there is no value in the Redux state for this field, **it will default to ''**. This is to ensure that the input is controlled. If you require that the value be of another type (e.g. Date or Number), you must provide initialValues to your form with the desired type of this field.
https://redux-form.com/8.2.0/docs/api/field.md/#-code-input-value-any-code-

I guess I will update my code to protect against this and only set the value when it’s not an empty string. Should I comment on #150 with this approach?

Reply:
i think commenting there would be helpful. we generally don't want to enable interface options that contradict the component's "contract". if consuming code deviates, "your mileage may vary"